source_account/source_variables.tf

```diff
@@ -0,0 +1,63 @@
+# General
+variable "region" {
+  description = "Region to deploy into"
```

Comment:
nit: but you could add types to all of these variables

Reply:
Updated